content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,123 @@
+---
+Title: Create a Flexible subscription
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/ 
+         /rc/administration/customize-pro.md 
+         /rc/administration/customize-flexible-plan/
+---
+
+When you create a subscription with a [Flexible plan]({{< relref "/rc/administration/setup/#subscription-plans" >}}), you customize it to your business needs.
+
+This means you:
```

Comment:
I feel like this wording is a little wonky but I can't think of a better alternative at the moment. Maybe "Customizing your subscription includes the following"? Or something along those lines?

Reply:
Fixed.